app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -502,6 +467,58 @@ class LuciBuildService {
     log.info('Published a request with ${buildRequests.length} builds');
   }
 
+  /// Create a Presubmit ScheduleBuildRequest using the [slug], [sha], and 
+  /// [checkName] for the provided [build] with the provided [checkRunId].
+  ScheduleBuildRequest _createPresubmitScheduleBuild({
+    required github.RepositorySlug slug,
+    required String sha,
+    required String checkName,
+    required Build build,
```

Comment:
Build and checkName are only passed on reruns, so I would avoid using them here. For example, you could require prNumber to be passed as well.

Then this would generalize for the other presubmit build scheduling

Reply:
Gotcha. Let me go ahead and make that change. Thank you.

Reply:
Done.